File: app/src/main/java/com/yoesuv/infomalangbatu/fragment/ChildFragmentThanks.java
package com.yoesuv.infomalangbatu.fragment;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.text.Html;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import com.yoesuv.infomalangbatu.R;

public class ChildFragmentThanks extends Fragment {

    private View v;
    private TextView tvThanks;

    public static ChildFragmentThanks getInstance(){
        return new ChildFragmentThanks();
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        v = inflater.inflate(R.layout.fragment_thanks, container, false);
        tvThanks = (TextView) v.findViewById(R.id.textView_terima_kasih);
        tvThanks.setText(Html.fromHtml(getResources().getString(R.string.trims)));

        return v;
    }
}

File: app/src/main/java/com/yoesuv/infomalangbatu/menu/other/views/ChildFragmentThanks.kt
package com.yoesuv.infomalangbatu.menu.other.views

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.databinding.DataBindingUtil
import androidx.fragment.app.Fragment
import com.yoesuv.infomalangbatu.R
import com.yoesuv.infomalangbatu.databinding.ChildFragmentThanksBinding
import com.yoesuv.infomalangbatu.utils.AppHelper

class ChildFragmentThanks : Fragment() {
    companion object {
        fun getInstance(): Fragment = ChildFragmentThanks()
    }

    private var binding: ChildFragmentThanksBinding? = null

    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?,
    ): View? {
        if (binding == null) {
            binding = DataBindingUtil.inflate(inflater, R.layout.child_fragment_thanks, container, false)
        }
        binding?.lifecycleOwner = viewLifecycleOwner
        return binding?.root
    }

    override fun onViewCreated(
        view: View,
        savedInstanceState: Bundle?,
    ) {
        super.onViewCreated(view, savedInstanceState)
        binding?.textViewThanks?.text = AppHelper.fromHtml(getString(R.string.trims))
    }
}
